src/sysupdate/sysupdate-resource.c

```diff
@@ -160,6 +173,29 @@ static int resource_load_from_directory(
         return 0;
 }
 
+static int resource_load_from_directory(
+                Resource *rr,
+                mode_t m) {
+        _cleanup_closedir_ DIR *d = NULL;
+
+        assert(rr);
+        assert(IN_SET(rr->type, RESOURCE_TAR, RESOURCE_REGULAR_FILE, RESOURCE_DIRECTORY, RESOURCE_SUBVOLUME));
+        assert(IN_SET(m, S_IFREG, S_IFDIR));
+
+        d = opendir(rr->path);
+
```

Comment:
we try to keep function calls and their immediate error handling together (and instead put sepating empty lines before the function call or after the error handline), hence please drop this empty line

Reply:
this hasn't been addressed btw